packages/components/src/components/post-icon/post-icon.tsx

```diff
@@ -17,14 +17,18 @@ export class PostIcon {
   private path: string;
   private loadedPath: string;
   private svgSource = '<svg viewBox="0 0 16 16"></svg>';
+  private svgElement: SVGElement;
+
+  @Element() host: HTMLPostIconElement;
 
   @State() pathForceCDN = false;
+  @State() svgStyles: string;
   @State() svgOutput: string;
 
   /**
    * The name of the animation (`cylon`, `cylon-vertical`, `spin`, `spin-reverse`, `fade`, `throb`).
    */
-  @Prop() readonly animation?: string;
+  @Prop() readonly animation?: string | null = null;
```

Comment:
By declaring the type of the animation property to `'cylon' | 'cylon-vertical' | 'spin' | 'spin-reverse' | 'fade' | 'throb'` it would directly set the story with a select control.

But I'm okay with the `string` type if you prefer.